libs/blocks/email-collection/email-collection.css

```diff
@@ -283,6 +289,22 @@
     overflow-y: auto;
     padding-right: var(--form-padding);
   }
+
+  .email-collection.no-image .foreground {
+    padding-left: calc(var(--form-padding) - var(--focus-outline-margin));
+  }
+
+  .email-collection .text > div > * {
+    margin-left: var(--focus-outline-margin);
```

Comment:
Might it make sense to replace `-left` with `-inline-start` to future proof this for RTL?